ocaml/xapi/import.ml

```diff
@@ -1273,9 +1273,10 @@ module VGPU : HandlerTools = struct
     | Found_VGPU _ ->
       handle_dry_run __context config rpc session_id state x precheck_result
     | Create vgpu_record -> begin
+        (* No need to check powerstate during VM import *)
         let vgpu = log_reraise "failed to create VGPU" (fun value ->
-            let vgpu = Client.VGPU.create ~rpc ~session_id ~vM:value.API.vGPU_VM ~gPU_group:value.API.vGPU_GPU_group
-                ~device:value.API.vGPU_device ~other_config:value.API.vGPU_other_config ~_type:value.API.vGPU_type in
+            let vgpu = Xapi_vgpu.create_with_powercheck ~__context ~device:value.API.vGPU_device ~gPU_group:value.API.vGPU_GPU_group
```

Comment:
I'm a afraid that this is not right. VGPU.create is done through the API, because it needs to be executed on the pool master. There is no way to add extra parameters.